app/services/darktide.server.ts

```diff
@@ -301,3 +188,23 @@ export async function getAccountTrait(auth: AuthToken, traitCategory: string) {
     }
   }
 }
+export async function getShopFor(auth: AuthToken, archetype: string, id: string) {
```

Comment:
```suggestion
export async function getCharacterStore(auth: AuthToken, characterArchetype: string, characterId: string) {
```
rename for consistency and clarity

fyi this function will get reused for melks store where the url changes from `credits_store` to something else `marks_store`. At that point the function arguments should become an object like 

```ts
type GetCharacterStoreArgs = {
	auth: AuthToken
	characterArchetype: string
	characterId: string
	store: "credits" | "marks"
}
export async function getCharacterStore({auth, characterArchetype, characterId, store}: GetCharacterStoreArgs)
```